src/libraries/Microsoft.Extensions.Configuration.Binder/src/Microsoft.Extensions.Configuration.Binder.csproj

```diff
@@ -5,7 +5,7 @@
     <EnableDefaultItems>true</EnableDefaultItems>
     <IsPackable>true</IsPackable>
     <EnableAOTAnalyzer>true</EnableAOTAnalyzer>
-    <ServicingVersion>3</ServicingVersion>
+    <ServicingVersion>4</ServicingVersion>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
```

Comment:
Nice, thanks for adding the change. I'll keep in mind when doing branding that GeneratePackageOnBuild was already set to true from the previous servicing release, so I don't reset it.